README.md

```diff
@@ -230,14 +209,43 @@ TBD
 
         $ poetry install
 
-### Run tests:
+### Run tests
 
         $ poetry run pytest
 
 ### Run tests when source changes (uses [pytest-watcher](https://github.com/olzhasar/pytest-watcher)):
 
         $ poetry run ptw . --now --clear
 
+### Releasing
+
+* Update the CHANGELOG to reflect the latest version and the
+  changes included in the release.
+
+* Show the current version and the possible next versions:
+
+        $ bump-my-version show-bump
+        0.3.0 ── bump ─┬─ major ─ 1.0.0
+                       ├─ minor ─ 0.4.0
+                       ╰─ patch ─ 0.3.1
+
+* Bump the version to the desired number, for example:
+
+        $ bump-my-version bump minor
+
+  You will see the latest commit & tag by looking at `git log`. You
+  can then push these to GitHub to trigger the CI/CD workflow.
```

Comment:
Wouldn't hurt to expressly say "you can now do `git push` to push the changes to GitHub and trigger..."

Reply:
Done